tests/h/views/client_test.py

```diff
@@ -41,7 +41,7 @@ def test_it_sets_custom_content_security_policy_header(self, pyramid_request):
 
         assert (
             csp_header
-            == "script-src 'self' https://cdn.hypothes.is https://www.google-analytics.com; style-src https://cdn.hypothes.is"
+            == "script-src 'self' https://cdn.hypothes.is https://www.google-analytics.com; style-src https://cdn.hypothes.is 'unsafe-inline'"
```

Comment:
Note that the single-quotes around `'self'` and `'unsafe-inline'` are intentional and required by the `Content-Security-Policy` header spec.